src/components/shared/WithSize.js

```diff
@@ -38,7 +50,7 @@ export const withSize = Wrapped =>
       window.removeEventListener('resize', this._resizeListener);
     }
 
-    _updateWidth(container) {
+    _updateWidth(container: *) {
```

Comment:
I think you should use HTMLElement here

Reply:
It's `null | Element | Text`, I refined the code to handle it.